src/main/javascript/src/views/outcome/OutcomeScoring.vue

```diff
@@ -96,14 +96,16 @@ import {mapActions, mapGetters} from 'vuex'
       participantScoreList() {
         let arr = []
         const scoresAssociatedwithOutcome = this.outcomeScores.filter((score) => score.outcomeId === this.outcome_id)
-        this.participantFilteredList.map(p=>{
+        let sortedparticipantFilteredList = this.participantFilteredList;
```

Comment:
this.participantFilteredList is already sorted, so I don't think it needs to be sorted again.

Reply:
yeah agreed. remove double sorting